app/parse.py

```diff
@@ -13,10 +22,108 @@ class Product:
     price: float
     rating: int
     num_of_reviews: int
+    additional_info: dict
+
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+_driver: WebDriver | None = None
+
+def get_driver() -> WebDriver:
+    return _driver
+
+
+def set_driver(new_driver: WebDriver) -> None:
+    global _driver
+    _driver = new_driver
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s: %(message)s]",
+    handlers=[
+        logging.FileHandler("parser.log"),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+
+def parse_hdd_block_proces(product_soup: BeautifulSoup) -> dict[str, float]:
```

Comment:
The function name `parse_hdd_block_proces` seems to have a typo. It should likely be `parse_hdd_block_process` to reflect the intended functionality more accurately.